CI/check-jsonschema.py

```diff
@@ -0,0 +1,90 @@
+import json
+from jsonschema import Draft7Validator
+from json_source_map import calculate
+from json_source_map.errors import InvalidInputError
+import os
+import sys
+
+json_data = {}
+json_string = ''
+schema = {}
+
+inputArgs = sys.argv
+errors = []
+
+
+def main():
+    if len(sys.argv) < 2:
+        print("JSON path required.")
+        return 1
+
+    for filename in inputArgs[1:]:
+        prep(filename)
+
+    with open('validation_errors.json', 'w') as outfile:
+        json.dump(errors, outfile)
+
+    if len(errors) > 0:
+        return 1
+
+    return 0
+
+
+def prep(filename):
+    global json_data
+    global json_string
+    global schema
```

Comment:
Rather than using globals, why not just assing them here and pass them to the next function (since they're not used anywhere else)?